addons/main.go

```diff
@@ -233,6 +236,13 @@ func enableClusterBootstrapAndConfigControllers(ctx context.Context, mgr ctrl.Ma
 		setupLog.Error(err, "unable to create CPIConfigController", "controller", "vspherecpi")
 		os.Exit(1)
 	}
+	if err := (&csicontroller.CSIConfigReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
```

Comment:
The log object should be passed in and used.

Reply:
it's being extracted from the context